packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.tsx

```diff
@@ -22,6 +22,13 @@ const WalletTransferFormAccountCard: React.FC<TProps> = ({ account, activeWallet
     const { isMobile } = useDevice();
     const isInput = type === 'input';
     const isModal = type === 'modal';
+    const badgeLabel = account?.demo_account ? 'virtual' : account?.landingCompanyName;
```

Comment:
@heorhi-deriv where we define property demo_account? I saw just is_virtual.

Reply:
hey @sergei-deriv we are receiving this property from the BE response in every transferable account when we do `transfer_between_accounts` request.